src/FSharpPlus/Control/Applicative.fs

```diff
@@ -28,7 +28,7 @@ type Apply =
     #if !FABLE_COMPILER
     static member        ``<*>`` (struct (f: Task<_>          , x: Task<'T>             ), _output: Task<'U>             , [<Optional>]_mthd: Apply) = Task.apply   f x : Task<'U>
     #endif
-    #if !NET45 && !NETSTANDARD2_0 && !FABLE_COMPILER
+    #if !NETSTANDARD2_0 && !FABLE_COMPILER
```

Comment:
Why do we keep netstandard ?

Reply:
We unfortunately need to keep netstandard in order to avoid compilation error for type level. Perhaps we can get some help from @cannorin  on this?